{"name": "登录", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'fail' == 'pass'\n  - fail\n  + pass", "trace": "self = <test_case.test_excel.TestExcel object at 0x000002BA42D79898>\nresult = {'data': {'account': 'Ediwin', 'password': '5184715875cdb3c620966d184e2f84d111b17e6327308d351f2cb187bb2d91c4', 'type': 0}, 'method': 'post', 'res': <Response [200]>, 'result': 'fail', ...}\n\n    @allure.severity(allure.severity_level.CRITICAL)  # 发生BUG时的严重程度\n    @pytest.mark.parametrize(\"result\", results)\n    def test_excel(self, result):\n        allure.dynamic.title(str(result['title']))\n        allure.dynamic.description(f\"{str(result['method'])} {str(result['url'])}\")\n        with allure.step(f\"method:{str(result['method'])}\"):\n            pass\n        with allure.step(f\"url:{str(result['url'])}\"):\n            pass\n        with allure.step(f\"data:{str(result['data'])}\"):\n            pass\n        with allure.step(f\"res:{str(result['res'].json())}\"):\n            pass\n        # print(111)\n        # print(result['res'].json().get('code'))\n        # print(222)\n>       assert result[\"result\"] == \"pass\"\nE       AssertionError: assert 'fail' == 'pass'\nE         - fail\nE         + pass\n\ntest_excel.py:38: AssertionError"}, "description": "post https://192.168.1.205/api/cmdb/user/login", "steps": [{"name": "method:post", "status": "passed", "start": 1609677258226, "stop": 1609677258226}, {"name": "url:https://192.168.1.205/api/cmdb/user/login", "status": "passed", "start": 1609677258226, "stop": 1609677258226}, {"name": "data:{'account': 'Ediwin', 'password': '5184715875cdb3c620966d184e2f84d111b17e6327308d351f2cb187bb2d91c4', 'type': 0}", "status": "passed", "start": 1609677258226, "stop": 1609677258226}, {"name": "res:{'code': 0, 'message': '', 'data': {'token': '5c320519-fc72-4445-94ea-5adc5d8140a2'}}", "status": "passed", "start": 1609677258226, "stop": 1609677258226}], "parameters": [{"name": "result", "value": "{'title': '登录', 'method': 'post', 'url': 'https://192.168.1.205/api/cmdb/user/login', 'data': {'account': 'Ediwin', 'password': '5184715875cdb3c620966d184e2f84d111b17e6327308d351f2cb187bb2d91c4', 'type': 0}, 'result': 'fail', 'res': <Response [200]>}"}], "start": 1609677258226, "stop": 1609677258226, "uuid": "3cda8c0f-ad4a-44da-bd2a-19a5f76f0079", "historyId": "12817c9e20c6ad6fa63e6dd8355bb363", "testCaseId": "c702837ae51103cc760768369bdfeabd", "fullName": "test_case.test_excel.TestExcel#test_excel", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_excel"}, {"name": "subSuite", "value": "TestExcel"}, {"name": "host", "value": "DESKTOP-T3GOI97"}, {"name": "thread", "value": "13148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_excel"}]}